src/iotile/models/properties.ts

```diff
@@ -0,0 +1,8 @@
+export class Properties {
+
+  constructor(datas) {
+    for (let data of datas) {
+      this[data.name] = data.value;
```

Comment:
Yes, that API is similar to the one you find in many Browser LocalStorage APIs or even the cookie API